src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -48,6 +48,21 @@ public HttpResponseMessage CreateCheckStatusResponse(HttpRequestMessage request,
             return this.config.CreateCheckStatusResponse(request, instanceId, this.attribute);
         }
 
+
+        /// <summary>
+        /// Creates an HTTP response for checking the status of the specified instance supporting synchronous response as well. 
+        /// </summary>
+        /// <param name="request">The HTTP request that triggered the current function.</param>
+        /// <param name="instanceId">The unique ID of the instance to check.</param>
+        /// <param name="totalTimeout">Total allowed timeout for output from the durable function</param>
+        /// <param name="retryTimeout">Timeout between checks for output from the durable function</param>
+        /// <returns>Task of <see cref="HttpResponseMessage"/></returns>
+        public async Task<HttpResponseMessage> CreateCheckStatusResponse(HttpRequestMessage request, string instanceId, int totalTimeout, int retryTimeout)
+        {
+            return await config.CreateCheckStatusResponse(request, instanceId, this.attribute, totalTimeout, retryTimeout);
```

Comment:
style nit: to be consistent with the rest of the codebase, please prefix all instance variables, instance properties, and instance methods with `this.`. There are several places in this PR where this needs to be updated.